### ISSUE TITLE
tag changed after MWM update is overwritten/reverted on upload to OSM

### ISSUE THREAD

**Issue by RedAuburn** [organicmaps/organicmaps#8102]
https://overpass-api.de/achavi/?changeset=150958957
see "Joven Club de Computación y Electrónica" (rightmost point)

[The name was changed 10 hours ago](https://www.openstreetmap.org/node/4818795057/history/4) to include "y Electronica", however the OM data is from a time before the change (240429). Changing an unrelated tag (adding `name:en`) has overwritten the name tag and reset it to the old name.

the editor should only upload changes the user has explicitly submitted.

**Comment by RedAuburn** [comment]
i think related to https://github.com/organicmaps/organicmaps/issues/6260

**Comment by biodranik** [comment]
Is it possible to reproduce this behavior on the osm dev server? Is that issue reproduces only in the latest master, or also in the previous build? There was another change related to name editing.

@kirylkaveryn @map-per can you please also take a look? It is related to name tag editing/duplicate removal/deleting name tags that were deleted by users.

**Comment by map-per** [comment]
> It is related to name tag editing/duplicate removal/deleting name tags that were deleted by users.

No, that's not related. The bug also happened in previous OM versions (I can't find the place where this happened any more though).
It looks like OM doesn't have any kind of "modified" flag to keep track of which values were edited by the user.
